### DISCUSSION TITLE
element3ds 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#4] [模板讨论]
element3ds 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
每日大礼包任务完成情况：【每日大礼包领取成功】
用户：【】 --- 元素币：【】 --- 活跃度：【】 --- 贡献值：【】 --- 超神：【】 --- 扩散：【】 --- 微金：【】 --- 智慧：【】 --- 神石：【】 --- 积分：【】

以上是签到失败的提示，实际显示成功，并显示上述信息。

**Reply by wjf0214** [reply]
我这正常，你这是不是遇到了盾？或者更新下cookie看看。

**Reply by miaoxiaozhao** [reply]
每日大礼包任务完成情况：【每日大礼包领取成功】
用户：【】 --- 元素币：【】 --- 活跃度：【】 --- 贡献值：【】 --- 超神：【】 --- 扩散：【】 --- 微金：【】 --- 智慧：【】 --- 神石：【】 --- 积分：【】

又遇到这个情况了，更新了cookies还是没用。看了一下，最后一次成功是在9月16日。
